### PR TITLE
Variance Metric From MONAI Core Integration to MONAI Label for all Apps

### PR DESCRIPTION
**Pull request by finalelement**
Integrating to the particular 'monailabel/tasks/scoring/epistemic_v2.py'

If there are other places where modifications to be made for scoring @SachidanandAlle please suggest.

Signed-off-by: Vishwesh Nath <vnath@nvidia.com>

### CODE REVIEW

File: monailabel/tasks/scoring/epistemic_v2.py [hunk 1]
```diff
@@ -84,14 +85,15 @@ def entropy_volume(self, vol_input):
 
     def variance_volume(self, vol_input):
         vol_input = vol_input.astype(dtype="float32")
+        variance_metric = VarianceMetric(include_background=True, threshold=0.0005, spatial_map=True)
 
+        variance = variance_metric(vol_input)
         # Threshold values less than or equal to zero
-        threshold = 0.0005
```
Comment (SachidanandAlle) [inline monailabel/tasks/scoring/epistemic_v2.py:89]: please remove the commented lines..

**Comment by SachidanandAlle** [comment]
doc build is failing because `from monai.metrics.active_learning_metrics import VarianceMetric ` is not available yet.. once monailabel starts using latest monai version then it should pass

**Comment by SachidanandAlle** [comment]
Depends on: https://github.com/Project-MONAI/MONAILabel/pull/979

File: monailabel/tasks/scoring/epistemic_v2.py (hunk 1 above)
Comment (finalelement) [reply monailabel/tasks/scoring/epistemic_v2.py:89]: Fixed

**Review by SachidanandAlle — APPROVED** [review]

**Pull request merged**